src/ProjectTemplates/test/EmptyWebTemplateTest.cs

```diff
@@ -32,7 +32,7 @@ public ITestOutputHelper Output
         }
 
         [ConditionalFact]
-        [SkipOnHelix("Cert failures", Queues = "All.OSX")]
+        [SkipOnHelix("Cert failure, https://github.com/dotnet/aspnetcore/issues/28090", Queues = $"All.OSX;{HelixConstants.Windows10Arm64};{HelixConstants.DebianArm64}")]
```

Comment:
Whoops, guess my plan of using const here didn't work like this. It would need to be a hard coded string with those three options.

Reply:
You can concat const strings with +